virtualization/windowscontainers/management/container_networking.md

```diff
@@ -235,13 +259,62 @@ C:\> docker run -it --network=MyTransparentNet --ip=10.80.123.32 windowsserverco
 
 ## Caveats and Gotchas
 
 ### Firewall
 
 The container host requires specific Firewall rules to be created to enable ICMP (Ping) and DHCP. ICMP and DHCP are required by Windows Server Containers to ping between two containers on the same host, and to receive dynamically assigned IP addresses through DHCP. In TP5, these rules will be created through the Install-ContainerHost.ps1 script. Post-TP5, these rules will be created automatically. All Firewall rules corresponding to NAT port forwarding rules will be created automatically and cleaned up when the container stops.
 
+### Existing vSwitch Blocking Transparent Network Creation
+
+When creating a transparent network, Docker creates an external vSwitch for the network then tries to bind the switch to an (external) network adapter - the adapter could be a VM Network Adapter or the physical network adapter. If a vSwitch has already been created on the container host, *and it is visible to Docker,* the Windows Docker engine will use that switch instead of creating a new one. However, if the vSwitch which was created out-of-band (i.e. created on the container host using HYper-V Manager or PowerShell) and is not yet visible to Docker, the Windows Docker engine will try create a new vSwitch and then be unable to connect the new switch to the container host external network adapter (because the network adapter will already be connected to the switch that was created out-of-band).
+
+For example, this issue would arise if you were to first create a new vSwitch on your host while the Docker service was running, then try to create a transparent network. In this case, Docker would not recognize the switch that you created and it would create a new vSwitch for the transparent network.
+
+There are three approaches for solving this issue:
+
+* You can of course delete the vSwitch that was created out-of-band, which will allow docker to create a new vSwitch and connect it to the host network adapter without issue. Before choosing this approach, ensure that your out-of-band vSwitch is not being used by other services (e.g. Hyper-V).
+* Alternatively, if you decide to use an external vSwitch that was created out-of-band, restart the Docker and HNS services to *make the switch visible to Docker.*
+```none
+PS C:\> restart-service hns
+PS C:\> restart-service docker
+```
+* Another option is to use the '-o' option to bind the transparent network (and the vSwitch created by Docker for the transparent network) to an alternate external network adapter on the container host--a network adapter other than the one being used by the vSwitch that was created out-of-band. The '-o' option is described above, in the [Transparent Network](https://msdn.microsoft.com/en-us/virtualization/windowscontainers/management/container_networking#transparent-network) section of this document.
```

Comment:
"use the '-o com.docker.network.windowsshim.interface' option to bind the transparent network's external vSwitch to a specific network adapter which is not already used." on the ocntainer host --a network adapter ...